packages/caliper-core/test/worker/rate-control/fixedFeedbackRate.js

```diff
@@ -144,5 +144,29 @@ describe('fixedFeedbackRate controller implementation', () => {
             sinon.assert.calledOnce(sleepStub);
             sinon.assert.calledWith(sleepStub, 20000);
         });
+
+        it('should sleep when no successful transactions have occurred', async () => {
+            txnStats.stats.txCounters.totalSubmitted = 5;
+            txnStats.stats.txCounters.totalFinished = 2;
+            txnStats.stats.txCounters.totalSuccessful = 0;
+            controller.generalSleepTime = 1;
+
+            await controller.applyRateControl();
+
+            sinon.assert.calledOnce(sleepStub);
+            sinon.assert.calledWith(sleepStub, 1 * controller.backOffTime);
+        });
+
+        it('should sleep when unfinished transactions are above the threshold', async () => {
+            txnStats.stats.txCounters.totalSubmitted = 44;
+            txnStats.stats.txCounters.totalFinished = 40;
+            controller.unfinishedPerWorker = 1;
+            controller.backOffTime = 100;
+
+            await controller.applyRateControl();
+
+            sinon.assert.calledOnce(sleepStub);
+            sinon.assert.calledWith(sleepStub, 88 * controller.backOffTime);
```

Comment:
this line, although the value is correct for the number sent to the sleep stub, it implies the backofTime is relevant to how the sleep value was calculated and thus the code path. It's in fact calculated as follows 
        let diff = (this.generalSleepTime * currentSubmitted - ((Date.now() - this.totalSleepTime) - this.stats.getRoundStartTime()));
and this is actually 44*200 = 8800

See also the comment about the description for further details